crates/linker/src/pe.rs

```diff
@@ -225,6 +225,15 @@ pub(crate) fn surgery_pe(executable_path: &Path, metadata_path: &Path, roc_app_b
     let mut data_bytes_added = 0;
     let mut file_bytes_added = 0;
 
+    // relocations between the sections of the roc application
+    // (as opposed to relocations for symbols the app imports from the host)
+    let inter_app_relocations = process_internal_relocations(
```

Comment:
I am surprised this is need. I thought I collected these relocation while appending sections to the app, but maybe I never did that. Has been a while since I dug through this code.

Reply:
this is the PE code; you've never touched this.

that said, there might be smarter way to do this? I just noticed that the relocations can "point" to sections that we did not yet process, so it doesn't fit into the existing flow. so what this does is pre-compute where the sections will go.

Reply:
Ah, I guess I didn't realize that I had switched to pe at that point. My bad.